code/index.html

```diff
@@ -1,22 +1,100 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
+    <!-- Meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Lauras magazine</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
+    <link href="https://fonts.googleapis.com/css2?family=Rajdhani:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Scope+One&display=swap" rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css" rel="stylesheet">
+
+
+</head>
+
+<body>
```

Comment:
There is nothing wrong with the code since it meets the requirements but I think it would have been easier/more controllable for you if you divided the content into sections.